File: server/frameworks/asp_net_core/json/Controllers/HomeController.cs
using Microsoft.AspNetCore.Mvc;
using System;

namespace aspnet_core.Controllers
{
    public class HomeController : Controller
    {
        [HttpGet("/")]
        public IActionResult Index()
        {
            var newObject = new
            {
                message = "Hello World",
                timestamp = DateTime.UtcNow.ToString("o"),
                randomNumber = new Random().Next(1000)
            };

            return Content($"<pre>{System.Text.Json.JsonSerializer.Serialize(newObject, new System.Text.Json.JsonSerializerOptions { WriteIndented = true })}</pre>", "text/html");
        }
    }
}

File: server/frameworks/laravel/json/app/Http/Controllers/HomeController.php
<?php

namespace App\Http\Controllers;

use Illuminate\Http\Request;

class HomeController extends Controller
{
    public function index()
    {
        $newObject = [
            'message' => 'Hello World',
            'timestamp' => now()->toISOString(),
            'randomNumber' => rand(0, 1000),
        ];

        return view('home', compact('newObject'));
    }
}
